tests/live/test_userid.py

```diff
@@ -98,22 +160,26 @@ def test_10_clear_registered_ip(self, fw, state_map):
         ips, tags = state.multi_register_02
         original = list(fw.userid.get_registered_ip())
         fw.userid.clear_registered_ip(ips[0], tags[0])
-        mod1 = list(fw.userid.get_registered_ip())
+        mod1 = fw.userid.get_registered_ip()
+        assert tags[0] not in mod1[ips[0]]
         fw.userid.clear_registered_ip(ips[0:4], tags[0:5])
-        mod2 = list(fw.userid.get_registered_ip())
+        mod2 = fw.userid.get_registered_ip()
+        assert all([all([tag not in mod2[ip] for tag in tags[0:5]]) for ip in ips[0:4]])
```

Comment:
Again, this test is too large.  Go ahead and split this stuff into multiple tests with appropriately descriptive names.